pandunia-lekse.csv

```diff
@@ -63,8 +63,8 @@ alofone||||allophone|allophone|alófono|||||||||||||alofono|allofoni (vaihtoehto
 alonamo||||that is (in other words, i.e., alias)||es decir (en otras palabras)|||||||||||||tio estas (alivorte)|eli (toisin sanoen)|innymi słowy|
 aloraya||||disagree||disentir (no estar de acuerdo)|discordar|||不同意|||||||||malkonsenti|olla eri mieltä|nie zgadzać się|
 aloraye||||disagreement (disconsent)||desacuerdo|||||||||||||||niezgoda|
+alojinsami||||heterosexual|hétérosexuel|heterosexual|||||||||||||samseksema|heteroseksuaali|heteroseksualny|
```

Comment:
Although this is a good word, I would like to keep on using -seksi in these. I wanted to redefine the root seks- to mean only sexual relationship or activity. Then it would work in this and other similar compounds.

Reply:
Okay. That sounds good to me.